src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -2596,3 +2596,12 @@
       - name: Disable an internal ingress gateway.
         text: az aks mesh disable-ingress-gateway --resource-group MyResourceGroup --name MyManagedCluster --ingress-gateway-type Internal
 """
+
+helps['aks ai'] = """
+    type: command
+    short-summary: Start a chat with the Azure Kubernetes Service expert.
+    long-summary: This command initiates a chat assistant with expertise in Azure Kubernetes Service, offering guidance on troubleshooting issues using az commands.
+    examples:
+        - name: How to create a AKS private cluster.
+          text: az aks ai (Prompt how to create a private aks cluster)
```

Comment:
As far as I know, cli command does not accept positional arguments. May be set an option for this?